### DISCUSSION TITLE
new maintainer

### DISCUSSION THREAD

**Discussion by sckott** [ropensci/bold#78] [General]
@devonorourke Since you've opened a few issues here, curious if you're interested in taking over maintainer role of the package? I can't maintain it anymore so looking for a replacement.

**Comment by devonorourke** [comment]
Thanks for the invitation @sckott, but unfortunately my current role prevents me from working on external projects. Not sure if you've corresponded with [Mike Robeson](https://uams-triprofiles.uams.edu/profiles/display/5680420) or [Nick Bokulich](https://hest.ethz.ch/en/research/professorships/person-detail.MjczNjI1.TGlzdC85MiwtNzMwNjY5NDYy.html) before, but I had a chance to work together with them on a COI project, [RESCRIPt](https://github.com/bokulich-lab/RESCRIPt), and they are both tremendously knowledgeable on issues of database construction and classification. Not sure if their interests are aligned with yours, but perhaps they can be a good sounding board for what other community members are interested in maintaining your fantastic BOLD R package.

**Reply by sckott** [reply]
Okay, understandable. 

Thanks very much for the tips! I'll try to follow up with those folks.
